config/kafka-monitor.properties

```diff
@@ -39,6 +39,9 @@
     "zookeeper.connect": "localhost:2181",
     "bootstrap.servers": "localhost:9092",
     "produce.record.delay.ms": 100,
+    "produce.topic.autoTopicCreationEnabled": true,
```

Comment:
With this config, when a new user of Kafka Monitor tries `./bin/kafka-monitor-start.sh config/kafka-monitor.properties` as suggested in our wiki on a cluster of only 1-2 machines, they will see topic creation failure without knowing that they need to change autoTopicReplicationFactor to make this work. This may be a frustrating experience. Maybe we should turn of autoTopicCreationEnabled here?